.env.dist

```diff
@@ -38,13 +38,19 @@ FIREBASE_SERVICE_ACCOUNT_PATH=/runtime/keys/firebase.json
 #+only-if(GOOGLE_CONTAINER_BUILDER_SOURCE_BUCKET):(GOOGLE_CONTAINER_BUILDER_ENABLED=true)
 #+only-if(GOOGLE_CONTAINER_BUILDER_PROJECT_ID):(GOOGLE_CONTAINER_BUILDER_ENABLED=true)
 #+only-if(GOOGLE_CONTAINER_BUILDER_SERVICE_ACCOUNT_PATH):(GOOGLE_CONTAINER_BUILDER_ENABLED=true)
+#+only-if(GOOGLE_CONTAINER_BUILDER_BUILD_TIMEOUT):(GOOGLE_CONTAINER_BUILDER_ENABLED=true)
+#+only-if(GOOGLE_CONTAINER_BUILDER_IMAGE):(GOOGLE_CONTAINER_BUILDER_ENABLED=true)
+#+only-if(GOOGLE_CONTAINER_BUILDER_VERSION):(GOOGLE_CONTAINER_BUILDER_ENABLED=true)
 #+file-to-propagate(GOOGLE_CONTAINER_BUILDER_SERVICE_ACCOUNT_PATH)
 #
 BUILDER_DEFAULT_ENGINE=docker
 
 GOOGLE_CONTAINER_BUILDER_ENABLED=false
 GOOGLE_CONTAINER_BUILDER_SOURCE_BUCKET=
 GOOGLE_CONTAINER_BUILDER_PROJECT_ID=
+GOOGLE_CONTAINER_BUILDER_BUILD_TIMEOUT=3600
+GOOGLE_CONTAINER_BUILDER_IMAGE=gcr.io/continuous-pipe-1042/cloud-builder
```

Comment:
Can you put the Docker Hub one here?

Reply:
I can, no need to stay BC with the old default value?

Reply:
No, it isn't working anyway as it was private 😓